examples/language-features/sets.qnt

```diff
@@ -25,8 +25,6 @@ module sets {
     n <= s.fold(0, (x, y) => x + y),
     s.in(powerset(s)),
     s == flatten(Set(s)),
-    // TODO: Enable after support for lists is added
-    // List(1, 1, 1).in(Set(1).allLists()),
```

Comment:
I don't see an `allLists` test in `lists.qnt`. Idk if we can add it already, but if not, it would be good to keep the comment as a reminder.

Reply:
We can’t support allLists in apalache currently.

I’ll follow up with Ana issue to track this.